niyodusengaclement/taskforce-todo-backend
# #9 Add an endpoint to change all status at the same time

#### What does this PR do?
 - add an endpoint to change all status at the same time
 
#### Type of change
---

> **Please select the relevant options**



#### Description of Task to be completed?
 - Add an endpoint to delete all todos
 - Add swagger documentation for this endpoint
 - Add unit test for this endpoint
#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/niyodusengaclement/taskforce-todo-backend.git), and `cd [the project name]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `npm install` to install the project dependencies
- Run `npm run migrate` to migrate new tables to the database
- Run `npm run dev` to start the server in development mode
- Create an account and login with your credentials
- Create more than one todos and then request this endpoint('/api/todos') with the `PUT` request ( add the received token when you were logging in in headers as value with this key `x-auth-token`)

#### Checklist:


#### What are the relevant issue(s)?
#9 

#### Screenshots (if appropriate)
